servant-client/test/Servant/ClientSpec.hs

```diff
@@ -30,7 +30,7 @@ module Servant.ClientSpec where
 import           Control.Applicative        ((<$>))
 #endif
 import           Control.Arrow              (left)
-import           Control.Concurrent         (forkIO, killThread, ThreadId)
+import           Control.Concurrent         (ThreadId, forkIO, killThread)
```

Comment:
There's still some formatting changes to imports like this one that I would remove. In some other places, too.